gcp_variant_transforms/vcf_to_bq_common.py

```diff
@@ -126,12 +144,12 @@ def read_headers(pipeline, pipeline_mode, known_args):
   return headers
 
 
-def get_merged_headers(headers, known_args):
-  # type: (pvalue.PCollection, argparse.Namespace) -> pvalue.PCollection
+def get_merged_headers(headers, known_args, allow_incompatible_records):
+  # type: (pvalue.PCollection, argparse.Namespace, bool) -> pvalue.PCollection
   """Applies the ``MergeHeaders`` PTransform on PCollection of ``VcfHeader``."""
```

Comment:
could you please add an Arg section? I think explaining `allow_incompatible_records` would be useful.